cmake.deps/cmake/MarkdownParserCMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.10)
+project(parser C)
```

Comment:
```suggestion
project(${PARSERLANG} C)
```

as a quick hack to avoid

```
CMake Warning:
  Manually-specified variables were not used by the project:

    PARSERLANG
```

Reply:
Actually `${PARSERLANG}` shouldn't be specified anywhere -- did I miss one?

Reply:
In `cmake.deps/cmake/BuildTreesitterParsers.cmake` we pass the variable `PARSERLANG` to `MarkdownParserCMakeLists.txt`, which then doesn't use it. It is possible to handle this in `MarkdownParserCMakeLists.txt` by not sending `PARSERLANG` to begin with with an `if()` but I figured this'd be more convenient for now.